docs/chainOps/k8s/tutorials/betanet.md

```diff
@@ -161,14 +191,15 @@ where:
 |`<chain_id>`|The Chain ID of the network (e.g.: sifchain).|
 |`<moniker>`|The moniker or name of your node as you want it to appear on the network.|
 |`<amount>`|The amount to stake, including the denomination (e.g.: 100000000rowan). The precision used is 1e18.|
-|`<gas>`|The gas price (e.g.: 0.5rowan).|
 |`<public_key>`|The public key of your validator (you got this in the previous step).|
+|`<gas>`| The per-transaction gas limit (e.g.: 300000rowan).|
```

Comment:
@sahilpant The gas does not include a denom, it's just simply provided as an integer (`e.g.: 300000`).